recipe/meta.yaml

```diff
@@ -66,9 +65,10 @@ build:
     {% if cuda_major == 11 %}
     - cusparselt  # [linux64]
     {% endif %}
-    #{% if cuda_major_minor >= (10, 1) %}
-    #- libcugraph  # [linux64]
-    #{% endif %}
+    {% if cuda_major_minor >= (10, 1) %}
+    - libcugraph  # [linux64]
```

Comment:
It appears `libcugraph` isn't included in the `run` requirements. Wonder if we might need to leave this out of this list to ensure it is included

Reply:
It is added in run_constrained as it's an optional dependency. If we decide to make cudnn optional to save disk usage, the same argument should be applied to libcugraph too as it's quite big.

Reply:
Ah ok. Yeah that makes sense